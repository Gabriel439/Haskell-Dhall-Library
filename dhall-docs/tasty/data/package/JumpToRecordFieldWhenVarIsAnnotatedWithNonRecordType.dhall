{-|
On this example, `dhall-docs` will detect that the variable `a` is annotated
with a non-record type, therefore not associating any information for jump to
definition i.e. hovering the `x` on the field-access expression won't highlight
anything
-}
let a : Bool = { x = "foo" }

in a.x
