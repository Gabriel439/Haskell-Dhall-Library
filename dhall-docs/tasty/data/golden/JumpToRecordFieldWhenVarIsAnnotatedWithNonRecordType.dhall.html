<!DOCTYPE HTML>
<html
  ><head
    ><title
    >/JumpToRecordFieldWhenVarIsAnnotatedWithNonRecordType.dhall</title
    ><link href="index.css" type="text/css" rel="stylesheet"
    /><link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap"
            type="text/css" rel="stylesheet"
    /><script src="index.js" type="text/javascript"
    /><meta charset="UTF-8"/></head
  ><body
    ><div class="nav-bar"
      ><img src="dhall-icon.svg" class="dhall-icon"
      /><p class="package-title"
      >test-package</p
      ><div class="nav-bar-content-divider"
      /><a id="switch-light-dark-mode" class="nav-option"
      >Switch Light/Dark Mode</a></div
    ><div class="main-container"
      ><h2 class="doc-title"
        ><span class="crumb-divider"
        >/</span
        ><a href="index.html"
        >test-package</a
        ><span class="crumb-divider"
        >/</span
        ><span href="index.html" class="title-crumb"
        >JumpToRecordFieldWhenVarIsAnnotatedWithNonRecordType.dhall</span></h2
      ><a data-path="/JumpToRecordFieldWhenVarIsAnnotatedWithNonRecordType.dhall"
          class="copy-to-clipboard"
        ><i
          ><small
          >Copy path to clipboard</small></i></a
      ><br
      /><div class="doc-contents"
        ><p
          >On this example, <code
          >dhall-docs</code
          > will detect that the variable <code
          >a</code
          > is annotated
with a non-record type, therefore not associating any information for jump to
definition i.e. hovering the <code
          >x</code> on the field-access expression won&#39;t highlight
anything</p>
</div
      ><h3
      >Source</h3
      ><div class="source-code"
        ><pre
          >{-|<br
          />On this example, `dhall-docs` will detect that the variable `a` is annotated<br
          />with a non-record type, therefore not associating any information for jump to<br
          />definition i.e. hovering the `x` on the field-access expression won&#39;t highlight<br
          />anything<br
          />-}<br
          />let <span id="var7-5" data-variable="var7-5"
                      class="variable-decl"
          >a</span
          > : Bool = { <span id="var7-18" data-variable="var7-18"
                             class="variable-decl"
          >x</span
          > = &quot;foo&quot; }<br
          /><br
          />in <a href="#var7-5" data-variable="var7-5" class="variable-use"
          >a</a
          >.x<br/></pre></div></div></body></html>